internal/config/cfg.go

```diff
@@ -31,12 +31,20 @@ type NotifierFrequency struct {
 type Config struct {
 	WeatherAPIKey string `envconfig:"WEATHER_API_KEY" required:"true"`
```

Comment:
I recommend splitting these variables by provider like this:

```suggestion
        Weather Weather
```

where `Weather` is defined as follows:

```go
type Weather struct {
    APIKey  string `envconfig:"WEATHER_API_KEY" required:"true"`
    BaseURL string `envconfig:"WEATHER_API_URL" required:"true"`
}
```

Reply:
should I create struct for every 3-rd party api settings?